jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/FormField.java

```diff
@@ -0,0 +1,322 @@
+                actualElement.input(value);
+                break;
+            case "textarea":
+                actualElement.input(value);
+                break;
+            case "mat-select":
+                actualElement.click();
+                MaterialSelectorContainer cdkOverlayContainer = new MaterialSelectorContainer();
+                cdkOverlayContainer.select(value);
+                break;
+            default:
+                break;
+        }
+    }
+
+    @JDIAction("Get value for '{name}' with index {0}")
+    public String value(int index) {
+        String value = "";
+        String type = getType(index);
+        WebElement element = getFormFields().get(index).find(By.xpath("//" + type));
+        UIElement actualElement = new UIElement(element);
+        switch (type) {
+            case "input":
+                value = actualElement.text(VALUE);
+                break;
+            case "textarea":
+                value = actualElement.text(VALUE);
+                break;
+            case "mat-select":
+                value = actualElement.getText();
+                break;
+            default:
+                break;
+        }
+        return value;
+    }
+
+    @JDIAction("Clear value from '{name}' with index {0}")
+    public void clear(int index) {
+        String type = getType(index);
+        WebElement element = getFormFields().get(index).find(By.xpath("//" + type));
+        UIElement actualElement = new UIElement(element);
+        actualElement.sendKeys(Keys.CONTROL + "a");
+        actualElement.sendKeys(Keys.DELETE);
+        focusOut();
+    }
+
+    @JDIAction("Click icon in '{name}' with index {0}")
+    public void clickIcon(int index) {
+        WebElement element = getFormFields().get(index).find(By.xpath("//mat-icon"));
+        UIElement actualElement = new UIElement(element);
+        actualElement.click();
+    }
+
+    @JDIAction("Get icon text in '{name}' with index {0}")
+    public String icon(int index) {
+        WebElement element = getFormFields().get(index).find(By.xpath("//mat-icon"));
+        UIElement actualElement = new UIElement(element);
+        return actualElement.getText();
+    }
+
+    @JDIAction("Get font attributes in '{name}' with index {0}")
+    public String font(int index) {
+        String type = getType(index);
+        WebElement element = getFormFields().get(index).find(By.xpath("//" + type));
+        UIElement actualElement = new UIElement(element);
+        return actualElement.getCssValue("font");
+    }
+
+    @JDIAction("Get color attribute in '{name}' with index {0}")
+    public String color(int index) {
+        String type = getType(index);
+        WebElement element = getFormFields().get(index).find(By.xpath("//" + type));
+        UIElement actualElement = new UIElement(element);
+        return actualElement.getCssValue("color");
+    }
+
+    @Override
+    public FormFieldsAssert is() { return new FormFieldsAssert().set(this); }
```

Comment:
Code convention is not respected. Please format this method.

Reply:
Updated.